### PR TITLE
[rocrand] Successful build in clean chroot

### PR DESCRIPTION
**Pull request by tpkessler**
Fix dependencies, install LICENSE to proper directory.

### CODE REVIEW

File: rocrand/PKGBUILD [hunk 1]
```diff
@@ -2,28 +2,29 @@
...
 pkgver=3.3.0
 pkgrel=1
-pkgdesc="RAND library for HIP programming language"
+pkgdesc='Pseudo-random and quasi-random number generator on ROCm'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocRAND"
 license=('MIT')
-makedepends=("hcc>=$pkgver" 'cmake')
-source=("https://github.com/ROCmSoftwarePlatform/rocRAND/archive/rocm-$pkgver.tar.gz")
+depends=('hip-hcc')
```
Comment (acxz) [inline rocrand/PKGBUILD:9]: Should this be just `hip` to allow for `hip-nvcc` and `hip-clang`?

File: rocrand/PKGBUILD [hunk 2]
```diff
@@ -2,28 +2,29 @@
...
+makedepends=('hcc' 'cmake' 'git')
+source=("$url/archive/rocm-$pkgver.tar.gz")
```
Comment (acxz) [inline rocrand/PKGBUILD:11]: Can we have ${pkgname}-${pkgver} to the source? See: https://wiki.archlinux.org/index.php/PKGBUILD#source

File: rocrand/PKGBUILD (hunk 1 above)
Comment (tpkessler) [reply rocrand/PKGBUILD:9]: rocrand works with ROCm only. For CUDA you have to use cuRAND which is part of CUDA.

File: rocrand/PKGBUILD (hunk 2 above)
Comment (tpkessler) [reply rocrand/PKGBUILD:11]: Good point. I've changed it.

File: rocrand/PKGBUILD [hunk 3]
```diff
@@ -2,28 +2,29 @@
 pkgname=rocrand
 pkgver=3.3.0
 pkgrel=1
-pkgdesc="RAND library for HIP programming language"
+pkgdesc='Pseudo-random and quasi-random number generator on ROCm'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocRAND"
```
[the remark this answers is not in the document]
Comment (tpkessler) [reply rocrand/PKGBUILD:7]: Same argument as before: official docu is just a collection of READMEs from the github repos. If we would like to change this, we should do it for all packages.

File: rocrand/PKGBUILD (hunk 1 above)
Comment (acxz) [reply rocrand/PKGBUILD:9]: I see kk

File: rocrand/PKGBUILD (hunk 3 above)
[the remark this answers is not in the document]
Comment (acxz) [reply rocrand/PKGBUILD:7]: since it is not important let us just tackle it as it comes up

File: rocrand/PKGBUILD [hunk 4]
```diff
@@ -2,28 +2,29 @@
 pkgname=rocrand
 pkgver=3.3.0
 pkgrel=1
```
Comment (acxz) [inline rocrand/PKGBUILD:4]: shoulddn't pkgrel be bumped?

**Comment by acxz** [comment]
Building clean chroot gt the following error:
```
CMake Error at /usr/share/cmake-3.17/Modules/CMakeFindDependencyMacro.cmake:47 (find_package):
  By not providing "Findamd_comgr.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "amd_comgr", but CMake did not find one.

  Could not find a package configuration file provided by "amd_comgr" with
  any of the following names:

    amd_comgrConfig.cmake
    amd_comgr-config.cmake

  Add the installation prefix of "amd_comgr" to CMAKE_PREFIX_PATH or set
  "amd_comgr_DIR" to a directory containing one of the above files.  If
  "amd_comgr" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  /opt/rocm/hip/lib/cmake/hip/hip-config.cmake:78 (find_dependency)
  cmake/VerifyCompiler.cmake:42 (find_package)
  CMakeLists.txt:50 (include)
```

File: rocrand/PKGBUILD (hunk 3 above)
[the remark this answers is not in the document]
Comment (tpkessler) [reply rocrand/PKGBUILD]: I've updated the URL.

File: rocrand/PKGBUILD (hunk 4 above)
Comment (tpkessler) [reply rocrand/PKGBUILD]: You're right. Done.

**Comment by tpkessler** [comment]
Did you install comgr in the clean chroot? I assume that hip-hcc depends on it, see my PR #157

**Comment by acxz** [comment]
I got it to work with using the latest hip-hcc 3.3.0-6. LGTM

**Pull request merged**